mp4/uuid.go

```diff
@@ -8,16 +8,19 @@ import (
 )
 
 const (
-	uuidTfxd = "\x6d\x1d\x9b\x05\x42\xd5\x44\xe6\x80\xe2\x14\x1d\xaf\xf7\x57\xb2"
-	uuidTfrf = "\xd4\x80\x7e\xf2\xca\x39\x46\x95\x8e\x54\x26\xcb\x9e\x46\xa7\x9f"
+	// UUIDTfxd - uuid for MSS tfxd box
+	UUIDTfxd = "\x6d\x1d\x9b\x05\x42\xd5\x44\xe6\x80\xe2\x14\x1d\xaf\xf7\x57\xb2"
+	// UUIDTfrf - uuid for MSS tfrf box
+	UUIDTfrf = "\xd4\x80\x7e\xf2\xca\x39\x46\x95\x8e\x54\x26\xcb\x9e\x46\xa7\x9f"
```

Comment:
These aren't strings, so I think it would be better to store these as `[16]byte`, or hex encoded. This will also make it better for people searching for these constants

Reply:
I played around a bit with real UUIDs and [16]byte, the code got quite a lot more complex, so I changed the documentation of the values so that the UUID values are in the comments and it is explained that the values are 16-byte string representations.